### DISCUSSION TITLE
Background Jobs Backing Store Choices

### DISCUSSION THREAD

**Discussion by kpconnell** [ServiceStack/Discuss#131] [Ideas]
Please allow the ability to use a central backing store (other RDBMS or Redis) for background jobs. 

SqlLite on a pet server?  LOL.

**Comment by mythz** [comment]
Background Jobs is designed specifically for SQLite which is designed to make use of SQLite-specific features and requires no external infrastructure dependencies.

> Also [SQLite on the server works great](https://servicestack.net/posts/scalable-sqlite).

---

ServiceStack v8.9 new [Database Jobs Feature](https://docs.servicestack.net/rdbms-background-jobs) now includes RDBMS support for PostgreSQL, SQL Server and MySQL
